parsec/parsec.c

```diff
@@ -1232,6 +1237,9 @@ int parsec_fini( parsec_context_t** pcontext )
     parsec_mempool_stats(context);
 #endif  /* PARSEC_PROF_TRACE */
 
+    (void) parsec_termdet_close_modules();
```

Comment:
Any reason why `parsec_termdet_close_modules` is not part of `parsec_termdet_fini`?